lib/stitches/generator_files/spec/features/api_spec.rb

```diff
@@ -80,7 +80,7 @@
       correct_code && correct_header
     end
 
-    failure_message_for_should do
+    failure_message do
```

Comment:
Does the gemspec require RSpec 3?  If not, it should or this will cause something to break downstream